src/components/dialog/useFocusTrap.js

```diff
@@ -35,14 +35,14 @@ export function useFocusTrap(ref: {current: HTMLDivElement | null}) {
       focusDescendant(dialogElement, isTabbingForward);
     }
 
-    document.addEventListener('keydown', handleKeydown);
-    document.addEventListener('keyup', handleKeyup);
-    document.addEventListener('focusin', handleFocusTrap);
+    dialogElement.addEventListener('keydown', handleKeydown);
+    dialogElement.addEventListener('keyup', handleKeyup);
+    dialogElement.addEventListener('focusin', handleFocusTrap);
```

Comment:
the `focusin` should fire after focusing element outside `dialogElement` so it won't trigger in this case.

Reply:
of course 🙏 thanks @bartoszlorek. I fixed it

Reply:
thanks, it looks ok